docs/users.md

```diff
@@ -522,6 +530,63 @@ vcluster-0                                          2/2     Running   0
 
 The nginx pod is the one with the name `nginx-x-default-x-vcluster`.
 
+## Working with an external control plane
+
+In this section, we will show an example of creating an external control plane to integrate
+a kind cluster named `ext1` into the same Docker network as your Kubeflex hosting cluster. This setup 
+requires that both clusters are accessible from each other within the shared Docker network.
```

Comment:
I find this confusing. It says that the purpose is "to integrate a kind cluster named `ext1` into the same Docker network as your Kubeflex hosting cluster" BUT it also says that they already have to be on the same Docker network. So the integration is already done?

I expected that the main issue here is the creation of a ControlPlane that represents a preexisting cluster.